src/app/button-page/button-page.component.ts

```diff
@@ -26,8 +28,12 @@ export class ButtonPageComponent implements OnInit {
     this.loadBoard();
   }
 
+  ngOnDestroy() {
+    this.subscription.unsubscribe();
```

Comment:
If we don't have a subscription (for some other error reason) this would fail.  But our global error handler would pick it up, it's not fatal, so it would log it and carry on....so it's probably ok isn't it?